doc/source/whatsnew/v0.19.0.txt

```diff
@@ -1005,6 +1034,7 @@ Bug Fixes
 - Bug in ``pd.read_csv()`` with ``engine='c'`` in which fields were not properly cast to float when quoting was specified as non-numeric (:issue:`13411`)
 - Bug in ``pd.read_csv``, ``pd.read_table``, ``pd.read_fwf``, ``pd.read_stata`` and ``pd.read_sas`` where files were opened by parsers but not closed if both ``chunksize`` and ``iterator`` were ``None``. (:issue:`13940`)
 - Bug in ``StataReader``, ``StataWriter``, ``XportReader`` and ``SAS7BDATReader`` where a file was not properly closed when an error was raised. (:issue:`13940`)
+- Bug in ``Series.unique()`` with datetime and timezone returns unique values without timezone, rather than return array of ``Timestamp`` with timezone (:issue:`13565`)
```

Comment:
I would put this also in the api changes section, as it was not a bug (it was a deliberate decision I think, but we changed our mind)